<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastroPacientes</class>
 <widget class="QDialog" name="cadastroPacientes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MEDICALSOFT</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #f0f0f0;
    border-radius: 10px;
}
QLabel {
    font-size: 14pt;
    color: #333;
}
QLineEdit, QDateEdit {
    border: 1px solid #ccc;
    border-radius: 5px;
    font-size: 12pt;
    background-color: #fff;
    color: #333;
    height: 30px;
}
QLineEdit {
    padding-left: 5px;
}
QDateEdit {
    padding-left: 8px;
}
QDateEdit::up-button, QDateEdit::down-button {
    background-color: #e0e0e0;
    border: none;
    width: 16px;
    height: 16px;
    padding: 0px;
}
QDateEdit::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;
}
QDateEdit::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right;
}
QDateEdit::up-arrow {
    image: url(:/icons/up-arrow.png);
    width: 12px;
    height: 12px;
}
QDateEdit::down-arrow {
    image: url(:/icons/down-arrow.png);
    width: 12px;
    height: 12px;
}
QPushButton {
    background-color: #e0e0e0;
    color: #333;
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 5px 10px;
    font-size: 12pt;
}
QPushButton#btnCancelarPac {
    background-color: #d0d0d0;
}
QPushButton:hover {
    background-color: #c0c0c0;
}
QPushButton#btnCancelarPac:hover {
    background-color: #b0b0b0;
}
QPushButton#btnCadastrarPac {
    background-color: #6db0ec;
}
QPushButton#btnCadastrarPac:hover {
    background-color: #5a9bd8;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtnome"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>CPF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtCpf"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Nascimento</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="txtData"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Diagnóstico (opcional)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtDiagn"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Convênio / Plano</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtconvenio"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>NºCelular</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtcontato"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>E-Mail</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtemail"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="btnCadastrarPac">
         <property name="text">
          <string>Cadastrar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancelarPac">
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
